txpipe/input_cats.py

```diff
@@ -157,16 +159,20 @@ def run(self):
                 for name in list(data.keys()):
                     data[name] = data[name][select]
 
-            # Simulate the various output data sets
-            mock_photometry = self.make_mock_photometry(data)
 
             # Cut out any objects too faint to be detected and measured.
             # We have to do this after the photometry, so that we know if
             # the object is detected, but we can do it before making the mock
             # metacal info, saving us some time simulating un-needed objects
-            self.remove_undetected(data, mock_photometry)
+            
+            #self.remove_undetected(data, mock_photometry)
```

Comment:
This function is not necessary anymore if we do not cut in SNR. Also it was giving me a an error, that's why I commented. I also had to move `self.make_mock_photometry(data)` a few lines below to avoid errors.